### DISCUSSION TITLE
Is it possible to integrate an editable table?

### DISCUSSION THREAD

**Discussion by fdion** [Budibase/budibase#7460] [Ideas]
For creating regular CRUD applications the Table + Create new through a form works as intended. The challenge comes when replacing spreadsheets. There doesn't appear to be an **editable version of the Table**. If it did, it could offer an UI similar to a spreadsheet. This supports the same operations but allow for a significantly faster data entry.

For example, looking at the templates, something like an Asset Manager, a Requirements Gathering or Manufacturing Quality Control, entering 50 records in a row is manageable in a grid format, but much more painful in a form based approach. Not only that, but it is easy to loose track of where one is and have to go back to the table view on a regular basis while entering data.

Tools like `Baserow` and `Nocodb` fill that spot, but do not address any significant multiscreen hierarchy (eg, create a product, then create quality requirements for that product). 

Having said that, if one is using a postgresql database, `nocodb` and `budibase` can see the same data, edit the same files. Assuming metadata is stable, is there a way to integrate `nocodb` in `budibase`?

**Comment by melohagan** [comment]
Hey @fdion 

This was something the team was discussing recently. We definitely would like a way for users to be more easily able to edit table data.

A NocoDB integration would be really cool! Maybe it'll be possible to build with the release of custom components at the end of September.

**Comment by fdion** [comment]
some progress:

Following https://docs.nocodb.com/setup-and-usages/share-view I used a private link for the table grid view. I used this URL using the `budibase` embed feature: https://docs.budibase.com/docs/embed

The table shows up, but it is not editable. I also haven't figured out yet how I can pass a default value (the parent). These issues are on the nocodb side.
